app.py

```diff
@@ -21,6 +42,20 @@
 cors = CORS(app)
 app.config["CORS_HEADERS"] = "Content-Type"
 
+#Suggestion route invoked by a POST request returning relevant JSON entries for suggestion values
+@app.route("/suggest")
```

Comment:
**[Required]** Missing some annotations here for CORS and error handling. Should be:
```python
@cross_origin()
@ErrorHandlerWrapper()
@app.route("/suggest")
```

Reply:
**[Required]** This endpoint needs to be below the `ErrorHandlerWrapper` class defintion. Put it right at the bottom of the file underneath the `knn_search` endpoint and above the `if __name == "__main__":` statement

Reply:
ok